info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-09-22T12:09:12.636Z"}
info: Connected to the database. {"timestamp":"2022-09-22T12:09:12.682Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-09-22T12:29:00.370Z"}
info: Connected to the database. {"timestamp":"2022-09-22T12:29:00.391Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'path' of undefined
    at _callee$ (C:\Users\soora\Desktop\test\awesome-project\src\controllers\/user.controller.js:14:26)
    at tryCatch (C:\Users\soora\Desktop\test\awesome-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\soora\Desktop\test\awesome-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\soora\Desktop\test\awesome-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\soora\Desktop\test\awesome-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\soora\Desktop\test\awesome-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\soora\Desktop\test\awesome-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\soora\Desktop\test\awesome-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at newUser (C:\Users\soora\Desktop\test\awesome-project\src\controllers\/user.controller.js:5:21)
    at Layer.handle [as handle_request] (C:\Users\soora\Desktop\test\awesome-project\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\soora\Desktop\test\awesome-project\node_modules\express\lib\router\route.js:144:13)
    at multerMiddleware (C:\Users\soora\Desktop\test\awesome-project\node_modules\multer\lib\make-middleware.js:13:41)
    at Layer.handle [as handle_request] (C:\Users\soora\Desktop\test\awesome-project\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\soora\Desktop\test\awesome-project\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (C:\Users\soora\Desktop\test\awesome-project\node_modules\express\lib\router\route.js:114:3) 
    url - /api/v1/users/student 
    method - POST 
    IP - ::1
   {"timestamp":"2022-09-22T14:38:57.594Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-09-22T15:07:14.773Z"}
info: Connected to the database. {"timestamp":"2022-09-22T15:07:14.822Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-09-22T15:09:22.828Z"}
info: Connected to the database. {"timestamp":"2022-09-22T15:09:22.875Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'filename' of undefined
    at _callee$ (C:\Users\soora\Desktop\test\awesome-project\src\controllers\/user.controller.js:14:68)
    at tryCatch (C:\Users\soora\Desktop\test\awesome-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\soora\Desktop\test\awesome-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\soora\Desktop\test\awesome-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\soora\Desktop\test\awesome-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\soora\Desktop\test\awesome-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\soora\Desktop\test\awesome-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\soora\Desktop\test\awesome-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at newUser (C:\Users\soora\Desktop\test\awesome-project\src\controllers\/user.controller.js:5:21)
    at Layer.handle [as handle_request] (C:\Users\soora\Desktop\test\awesome-project\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\soora\Desktop\test\awesome-project\node_modules\express\lib\router\route.js:144:13)
    at multerMiddleware (C:\Users\soora\Desktop\test\awesome-project\node_modules\multer\lib\make-middleware.js:13:41)
    at Layer.handle [as handle_request] (C:\Users\soora\Desktop\test\awesome-project\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\soora\Desktop\test\awesome-project\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (C:\Users\soora\Desktop\test\awesome-project\node_modules\express\lib\router\route.js:114:3) 
    url - /api/v1/users/student 
    method - POST 
    IP - ::1
   {"timestamp":"2022-09-22T15:24:27.282Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-09-22T16:04:43.929Z"}
info: Connected to the database. {"timestamp":"2022-09-22T16:04:43.952Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'filename' of undefined
    at _callee$ (C:\Users\soora\Desktop\test\awesome-project\src\controllers\/user.controller.js:14:68)
    at tryCatch (C:\Users\soora\Desktop\test\awesome-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\soora\Desktop\test\awesome-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\soora\Desktop\test\awesome-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\soora\Desktop\test\awesome-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\soora\Desktop\test\awesome-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\soora\Desktop\test\awesome-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\soora\Desktop\test\awesome-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at newUser (C:\Users\soora\Desktop\test\awesome-project\src\controllers\/user.controller.js:5:21)
    at Layer.handle [as handle_request] (C:\Users\soora\Desktop\test\awesome-project\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\soora\Desktop\test\awesome-project\node_modules\express\lib\router\route.js:144:13)
    at multerMiddleware (C:\Users\soora\Desktop\test\awesome-project\node_modules\multer\lib\make-middleware.js:13:41)
    at Layer.handle [as handle_request] (C:\Users\soora\Desktop\test\awesome-project\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\soora\Desktop\test\awesome-project\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (C:\Users\soora\Desktop\test\awesome-project\node_modules\express\lib\router\route.js:114:3) 
    url - /api/v1/users/student 
    method - POST 
    IP - ::1
   {"timestamp":"2022-09-22T16:04:56.027Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-09-22T16:05:28.140Z"}
info: Connected to the database. {"timestamp":"2022-09-22T16:05:28.160Z"}
error: 
    status - 500 
    message - TypeError: Cannot read property 'path' of undefined
    at _callee$ (C:\Users\soora\Desktop\test\awesome-project\src\controllers\/user.controller.js:14:68)
    at tryCatch (C:\Users\soora\Desktop\test\awesome-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\soora\Desktop\test\awesome-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\soora\Desktop\test\awesome-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\soora\Desktop\test\awesome-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\soora\Desktop\test\awesome-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\soora\Desktop\test\awesome-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\soora\Desktop\test\awesome-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at newUser (C:\Users\soora\Desktop\test\awesome-project\src\controllers\/user.controller.js:5:21)
    at Layer.handle [as handle_request] (C:\Users\soora\Desktop\test\awesome-project\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\soora\Desktop\test\awesome-project\node_modules\express\lib\router\route.js:144:13)
    at multerMiddleware (C:\Users\soora\Desktop\test\awesome-project\node_modules\multer\lib\make-middleware.js:13:41)
    at Layer.handle [as handle_request] (C:\Users\soora\Desktop\test\awesome-project\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\soora\Desktop\test\awesome-project\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (C:\Users\soora\Desktop\test\awesome-project\node_modules\express\lib\router\route.js:114:3) 
    url - /api/v1/users/student 
    method - POST 
    IP - ::1
   {"timestamp":"2022-09-22T16:05:58.143Z"}
